### ISSUE TITLE
Find more descriptive name for this project

### ISSUE THREAD

**Issue by sickill** [asciinema/asciicast2gif#14]
a2gif can mean anything2gif.

My original (and obvious) idea for the name of this tool was asciicast2gif. However there's already [pettarin/asciicast2gif](https://github.com/pettarin/asciicast2gif) (it does the same thing but takes a different approach).

**Comment by pettarin** [comment]
Hi, I am more than happy to let the name ``asciicast2gif`` go, if you want to take it.

(Yes, I chose it because it is <FORMAT>2<FORMAT> and I think it is better than <PROGRAM>2<FORMAT> as in tav's name.)

I created my project because I needed to convert to GIF, but I could not make tav's code to run (I do not remember the details) and I wanted to perform the thing locally. I am not really going to develop it further. Since it is not "distributed" anywhere (npm, etc.), I think there is no chance for confusing the two.

I have also just added in my README a link to your project (this repo), since users will be better off using your tool than mine.
